src/diffpy/structure/atom.py

```diff
@@ -215,6 +285,22 @@ def xyz_cartn(self):
 
     @xyz_cartn.setter
     def xyz_cartn(self, value):
+        """
+        Set the absolute Cartesian coordinates at `xyz`.
+
+        Setting ``xyz_cartn`` or any of its items such as ``xyz_cartn[0]``
+        updates the fractional coordinates `xyz`.
+
+        Parameters
+        ----------
+        value: numpy.ndarray
+            The value of fractional coordinates within the associated `lattice` to be set.
+
+        Returns
+        -------
+        None.
+        """
+
```

Comment:
Property setter function should not be documented because its docstrings are not displayed.  According to the sphinx-napoleon doc any notable behavior of a property setter should be included in the getter (`xyz_cartn(self)`) docstring.

Please merge with the getter docstring.